source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RichTextBlockRenderer.java

```diff
@@ -188,49 +180,19 @@ else if ((richTextBlock = RichTextBlock.dynamic_cast(baseCardElement)) == null)
         }
 
         // RichTextBlock properties
-        // Wrap
-        // MaxLines
         // HorizontalAlignment
-        // Paragraphs
-        if (!richTextBlock.GetWrap())
-        {
-            textView.setMaxLines(1);
-        }
-
-        int maxLines = (int)richTextBlock.GetMaxLines();
-        if (maxLines > 0 && richTextBlock.GetWrap())
-        {
-            textView.setMaxLines(maxLines);
-        }
-        else if (!richTextBlock.GetWrap())
-        {
-            textView.setMaxLines(1);
-        }
+        // Inlines
 
         textView.setGravity(TextRendererUtil.getTextAlignment(richTextBlock.GetHorizontalAlignment()));
 
         // This is the section for rendering the paragraphs
         // Every paragraph may contain contains any number of inlines
         // The current inline element types are TextRun
-        ParagraphVector paragraphs = richTextBlock.GetParagraphs();
-        int paragraphCount = (int)paragraphs.size();
+        InlineVector inlines = richTextBlock.GetInlines();
 
         textView.setText("");
-
-        for (int i = 0; i < paragraphCount; ++i)
-        {
-            Paragraph p = paragraphs.get(i);
-            InlineVector inlines = p.GetInlines();
-
-            if ((i != 0) && (inlines.size() != 0))
-            {
-                textView.append(System.getProperty("line.separator"));
-            }
-
-            SpannableStringBuilder convertedString = buildSpannableParagraph(renderedCard, inlines, cardActionHandler, hostConfig, renderArgs);
-
-            textView.append(convertedString);
-        }
+        SpannableStringBuilder convertedString = buildSpannableParagraph(renderedCard, inlines, cardActionHandler, hostConfig, renderArgs);
```

Comment:
>buildSpannableParagraph [](start = 49, length = 23)

Does this change include removing markdown support for rich text blocks?

Reply:
now it does :D

---
In reply to: [275498255](https://github.com/Microsoft/AdaptiveCards/pull/2688#discussion_r275498255) [](ancestors = 275498255)